gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/Start.groovy

```diff
@@ -0,0 +1,4 @@
+package com.novoda.gradle.command
+
+class Start extends Run {
```

Comment:
We could wire a stop task alias without the extra class
Am 25.06.2015 18:20 schrieb "Volker Leck" <notifications@github.com>:

> In
> gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/Start.groovy
> <https://github.com/novoda/gradle-android-command-plugin/pull/78#discussion_r33273902>
> :
>
> > @@ -0,0 +1,4 @@
> > +package com.novoda.gradle.command
> > +
> > +class Start extends Run {
>
> Might make sense to have a start alias for run. Should also be wire by
> default, if we want to go with it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/novoda/gradle-android-command-plugin/pull/78/files#r33273902>
> .
>